flink-connectors/flink-connector-jdbc/src/test/java/org/apache/flink/connector/jdbc/table/JdbcLookupTableITCase.java

```diff
@@ -120,7 +120,7 @@ public void testLookup() throws Exception {
 				.build());
 		if (useCache) {
```

Comment:
Would be better as following,

> 	JdbcLookupOptions.Builder lookupOptionsBuilder = JdbcLookupOptions.builder().setMaxRetryTimes(0);
		if (useCache) {
			lookupOptionsBuilder.setCacheMaxSize(1000).setCacheExpireMs(1000 * 1000);
		}
		builder.setLookupOptions(lookupOptionsBuilder.build());

Reply:
Thank you for your careful review, I have fixed the code